src/QueryComplexity.ts

```diff
@@ -33,6 +33,7 @@ import {
 export type ComplexityEstimatorArgs = {
   type: GraphQLCompositeType,
   field: GraphQLField<any, any>,
+  node: FieldNode | FragmentDefinitionNode | InlineFragmentNode | OperationDefinitionNode,
```

Comment:
This can only be `FieldNode` so we can narrow this interface. If we ever need to support the other nodes, we probably need a different approach (they don't have `args` or a `field` that would make sense).

Reply:
Oh whoops, no that's a mistake. I totally misread the code, my bad. I don't need the parent node, just the calculated node.